package.json

```diff
@@ -25,6 +25,9 @@
       "git add"
     ]
   },
+  "optionalDependencies": {
+    "@ckb-lumos/indexer": "0.4.5"
```

Comment:
lumos version should greater v0.5.1, 0.4.5 packed ckb-indexer have index bug.

Reply:
> lumos version should greater v0.5.1, 0.4.5 packed ckb-indexer have index bug.

v0.5.1 cannot be installed for the moment but v0.4.5 can be.
But v0.4.5 doesn't matter because lumos will update frequently so the version changes frequently and it's not used in the sdk, just used in an example.

Reply:
Got it, but I still recommend upgrade lumos indexer version later